tests/common/protocol/test_imds.py

```diff
@@ -386,99 +386,86 @@ def test_endpoint_fallback(self):
         # http GET calls and enforces a single GET call (fallback would cause 2) and
         # checks the url called.
 
-        test_subject = imds.ImdsClient("foo.bar")
+        test_subject = imds.ImdsClient()
```

Comment:
We should probably change this test name + comment since we're no longer using a fallback method.